onnx/backend/test/case/node/__init__.py

```diff
@@ -56,11 +56,15 @@ def function_expand_helper(node,  # type: NodeProto
         for internal_name in internal_node.input:
             if internal_name in io_names_map:
                 new_node.input.append(io_names_map[internal_name])
+            elif internal_name in node.input:
```

Comment:
I don't understand this. The name of the actual parameter in the call-site node is irrelevant here. Lines 57-58 seem to do the correct thing here.

Reply:
I guess it is a leftover from trying to troubleshoot the function expansion. I removed it now